src/backend/core/factories.py

```diff
@@ -75,7 +91,7 @@ class Meta:
 
     title = factory.Sequence(lambda n: f"document{n}")
     excerpt = factory.Sequence(lambda n: f"excerpt{n}")
-    content = factory.Sequence(lambda n: f"content{n}")
+    content = YDOC_HELLO_WORLD_BASE64
```

Comment:
To be consistent maybe `excerpt` can be computed from this content ?

Reply:
bof. That's just a factory. KISS?